lib/web/templates/batch/show.html.eex

```diff
@@ -0,0 +1,29 @@
+<main role=main><div class=wrapper>
+
+<h2 class=header>Batch Details</h2>
+
+<%= if @batch do %>
+<p>Priority: <%= @batch.priority %></p>
+<p>State: <%= stringify_state(@batch.state) %></p>
+<p>
+  Status:
+  <%= for status <- @statuses do %>
+    <%= if status.url do %>
+    <a href="<%= status.url %>"><%= status.identifier %></a>
+    <% else %>
+    <span><%= status.identifier %></span>
+    <% end %>

```

Comment:
One thing that seems like it would be really useful here would be to show the state of individual statuses. These are recorded in the database, so it should be easily doable by adding a line like this to the end:

```iex
(<%= stringify_state(status.state) %>)
```

So that it'll report something like `ci/buildkite (Running)`